lookout/style/format/benchmarks/general_report.py

```diff
@@ -307,14 +294,19 @@ def generate_model_report(self) -> str:
         """
         return generate_model_report(model=self.model)
 
-    def generate_report(self, fixes: Iterable[FixData]) -> str:
+    def generate_quality_report(self, fixes: Iterable[FixData]) -> str:
         """
         Generate quality report: classification report, confusion matrix, files with most errors.
 
         :return: report.
         """
-        return generate_report(chain.from_iterable(fix.all_vnodes for fix in fixes),
-                               self.config["n_files"])
+        fixes = list(fixes)
```

Comment:
Why do we need it? Remove it.

Reply:
It can be memory-intence

Reply:
We need it because we cannot take the first element from an `Iterable`